src/ontology/mondo-edit.obo

```diff
@@ -520739,6 +520724,32 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
 property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
 
+[Term]
+id: MONDO:0800453
+name: juvenile absence epilepsy
+def: "Juvenile absence epilepsy (JAE) is a genetic epilepsy with onset occurring around puberty. JAE is characterized by sporadic occurrence of absence seizures, frequently associated with a long-life prevalence of generalized tonic-clonic seizures (GTCS) and sporadic myoclonic jerks." [Orphanet:1941]
+subset: gard_rare {source="GARD:2162"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:1941"}
+subset: orphanet_rare {source="Orphanet:1941"}
+subset: predisposition
```

Comment:
do not add this subset, especially since this term is not a "predisposition", it is an actual disease.

Reply:
fixed with cf4b0ef5f